### DISCUSSION TITLE
OpenDataCam meets R (German Only)

### DISCUSSION THREAD

**Discussion by vsaw** [opendatacam/opendatacam#508] [Show and tell]
![opendatacamcounterid](https://user-images.githubusercontent.com/829308/145606509-40a08402-4095-4e02-9582-8704090f1246.jpg)

Just found this blog post by @yauh how he was running ODC to count more thank 5k cars pass his home on a single day!  🤯
https://www.herrhochhaus.de/OpenDataCam-meets-R/

Thanks for blogging @yauh!

**Comment by yauh** [comment]
Thanks for bringing it up here :)
And should you know some R but no German, check out the Repo and maybe add your own plots, if you like ;)
https://github.com/yauh/opendatacam-statistics
